controller/sp-validator/tmpl/validating_webhook_configuration.go

```diff
@@ -0,0 +1,22 @@
+package tmpl
+
+// ValidatingWebhookConfigurationSpec provides a template for a
+// ValidatingWebhookConfiguration.
+var ValidatingWebhookConfigurationSpec = `
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: {{ .WebhookConfigName }}
+webhooks:
```

Comment:
Could you clarify the intention? If I understand correctly this would mean that if there's a timeout or a program error the service profile will pass validation regardless.
If this is set as a parameter in the the webhook config yaml, where would the parameter value come from?

Reply:
`Ignore` is the default. The other supported value is `Fail`. The result of the service profile validation is decided by the `Allow` flag in the admission response. I was thinking the value can come from a new flag added to the server.

Let's just omit this flag for now. We can re-visit this later, as it's relevant to the other webhooks too.